src/extension/geopkg-output/src/main/java/org/geoserver/tiles/AbstractTilesGetMapOutputFormat.java

```diff
@@ -459,6 +493,19 @@ protected GridSubset findBestGridSubset(GetMapRequest req) {
         return gridSubsets.iterator().next();
     }
 
+    // finds the appropriate gridset zoom levels for the request
```

Comment:
Why are all these (across the PR) common comments, rather than javadocs?
It's valid but weird at the same time.

Reply:
David is gradually adjusting coding style.  I am focused on the public methods (not overrides) when I reviewed.

Reply:
Right, but this is still supported code... looks like it was written by someone whose first language is not Java...